l10n.js

```diff
@@ -219,40 +223,70 @@ document.webL10n = (function(window, document, undefined) {
               parsedRawLinesCallback();
               return;
             }
-            var line = entries.shift();
```

Comment:
With regards to ```Comments after a EOL. Are they part of the previous line or should they be ignored``` From my reading it appears that comments after an EOL are also not supported - it appears a comment is a line that begins with ```#``` So this check should not be needed?